compiler/rustc_session/src/options.rs

```diff
@@ -150,6 +151,9 @@ top_level_options!(
 
         target_triple: TargetTriple [TRACKED],
 
+        /// Effective logical environment used by `env!`/`option_env!` macros
+        logical_env: FxHashMap<String, String> [UNTRACKED],
```

Comment:
I think this should be tracked - if the input changes, the output may change as well.

Reply:
also raced :D